src/pages/playground/index.tsx

```diff
@@ -22,10 +24,26 @@ import { trpc } from 'lib/client/trpc';
 
 import type PageWithLayout from 'types/persistentLayout';
 
+const isValidPlaygroundRequestCategoryValue = (
+  value: string | string[] | undefined
+): value is PlaygroundRequestCategory[] => {
+  if (value === undefined) return false;
+  if (Array.isArray(value)) {
+    return value.some((category) => category in PlaygroundRequestCategory);
+  }
+  return value in PlaygroundRequestCategory;
+};
+
 const Playground: PageWithLayout = ({}) => {
+  const router = useRouter();
+
   const [filters, setFilters] = useState<
     trpc['playground']['getAllRequests']['input']
   >(() => ({
+    categories: isValidPlaygroundRequestCategoryValue(router.query.category)
+      ? router.query.category
+      : undefined,
+    isPaidRequest: router.query.isPaidRequest === 'true',
```

Comment:
The "Volunteer" filter value seems to be `false` when `isPaidRequest` is `undefined`.
This could be solved like you did in the other file:
`
    ...(router.query.isPaidRequest && {
      isPaidRequest: router.query.isPaidRequest === 'true',
    }),
`